mlrun/datastore/inmem.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base import DataStore, FileStats
+
+
+class InMemStore(DataStore):
```

Comment:
Characters don't cost money, suggest to change it to `in_memory.py` file, `InMemoryStore` class, `in_memory_store` variable names etc...

Reply:
made it longer